src/collections/_documentation/workflow/search.md

```diff
@@ -37,6 +37,22 @@ The tokens `is:resolved` and `user.username:"Jane Doe"` are standard search toke
 
 The token `example error` is utilizing the optional raw search and is passed as part of the issue search query (which uses a CONTAINS match similar to SQL). When using the optional raw search, you can provide _one_ string, and the query uses that entire string.
 
+### Using `OR` & `AND` Conditions
+
+Use `OR` and `AND` between tokens, and use parentheses `()` to group conditions. `AND` can also be used between non-aggregates and aggregates. However, `or` cannot. 
```

Comment:
I would add an alert box or something prominent here about which products support OR/AND, because this page starts off talking about search in general. in particular, the most popular search bar is the issue stream which does not support it

Reply:
Agreed. @MimiDumpling this new search synatax is only supported on the Performance monitoring product features. Specifically, this includes Discover, the Performance tab, and Metric Alerts.

Reply:
Made this Note in the new branch: https://github.com/getsentry/sentry-docs/pull/1909